README.md

```diff
@@ -1,10 +1,86 @@
 # CARTA Frontend
+The CARTA frontend is used in conjunction with the [CARTA backend](https://github.com/CARTAvis/carta-backend), and for general use a pre-built production frontend is usually installed alongside the backend automatically. 
+However, a user may wish to install the frontend component separately in order to test beta versions, for example, or to develop it.
```

Comment:
This should either have an extra newline or one fewer one. I think it's fine to keep this sentence in the same paragraph.